### DISCUSSION TITLE
Switching between pages getting "[HTTP]:500 - [CorrelationId]:e17f3aa1-a0e4-9000-5549-35a31fdae483 [Version]:16.0.0.25019" error.

### DISCUSSION THREAD

**Discussion by harigoudA** [microsoft-search/pnp-modern-search#3866] [Q&A]
Hi,
I am using "PnP Modern Search - Search Web Parts - v4" Version 4.11.0 in SharePoint online Search page. I am encountering an issue while switching pages using pagination.

Please find the below screen shot for the same.

![500 correaltion issue](https://github.com/user-attachments/assets/6e479872-4a6b-4bd0-994b-2e67f6d57aec)

**Comment by kasperbolarsen** [comment]
Any chance you can upgrade to the current version? We can only support the most current version

**Comment by wobba** [comment]
In general 500 errors are due to web part misconfigurations like using a non-refinable property as a filter/sorting or random API issues.

**Comment by harigoudA** [comment]
Thanks for your reply @wobba , @kasperbolarsen .
I have figured out the issue, we are not getting this issue if we don't apply sorting for "Title" and "Modified Date". Once we apply sorting we are getting this issue.
Reason: The issue is with the ‘Title’ and ‘Modified Date’ crawled properties. By default, there is no sort option for these two items. However, we have added these fields as ‘RefinableString’ and made them sortable after resolving this issue.

**Reply by wobba** [reply]
You can sort on the LastModifiedTime managed property. (crawled properties are for data mapping and not directly used in search 🙂)

**Reply by harigoudA** [reply]
Hello @wobba,
I have tried "LastModifiedTime" on documents, if we use this managed property we are not getting the exact modified date time.
Ex: I have an old document from the year 2022. When I upload the document to our document library, the modified date should reflect the latest update date and time. However, it is currently showing the year 2022.
Please refer the below image for the same.
![image](https://github.com/user-attachments/assets/44be3678-e380-456b-a91b-ca8e67a0df5a)

**Reply by wobba** [reply]
Indeed and is one of the nuances of dates. If you migrate/upload content using a propert tool it will retain the created/modified dates and stamp them into SharePoint. Or you can do so yourself using for example PowerShell.
